ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/processed/VariableDataProvider.kt

```diff
@@ -50,6 +50,9 @@ internal class VariableDataProvider(
     }
 
     fun periodName(rcPackage: Package): String? {
+        if (rcPackage.packageType == PackageType.CUSTOM) {
```

Comment:
Ditto to iOS: `UNKNOWN` should use this too.

Reply:
updated here as well, thanks!